rosbag2_tests/test/rosbag2_tests/test_rosbag2_info_end_to_end.cpp

```diff
@@ -68,6 +68,56 @@ TEST_P(InfoEndToEndTestFixture, info_end_to_end_test) {
       "Serialization Format: cdr"));
 }
 
+TEST_P(InfoEndToEndTestFixture, info_with_verbose_option_end_to_end_test) {
+  internal::CaptureStdout();
+  auto exit_code = execute_and_wait_until_completion(
+    "ros2 bag info bag_with_topics_and_service_events --verbose",
+    bags_path_);
+  std::string output = internal::GetCapturedStdout();
+  auto expected_storage = GetParam();
+  auto expected_file = rosbag2_test_common::bag_filename_for_storage_id(
+    "bag_with_topics_and_service_events", GetParam());
+  std::string expected_ros_distro = "unknown";
+
+  EXPECT_THAT(exit_code, Eq(EXIT_SUCCESS));
+  // The bag size depends on the os and is not asserted, the time is asserted time zone independent
+  EXPECT_THAT(
+    output, ContainsRegex(
+      "\nFiles:             " + expected_file +
+      "\nBag size:          .*B"
+      "\nStorage id:        " + expected_storage +
+      "\nROS Distro:        " + expected_ros_distro +
+      "\nDuration:          0\\.70s"
+      "\nStart:             Nov  7 2023 16:30:36\\..* \\(1699345836\\..*\\)"
+      "\nEnd:               Nov  7 2023 16:30:36\\..* \\(1699345836\\..*\\)"
```

Comment:
@Barry-Xu-2018 We've got a failure in these two lines again. I think the problem is in different time zones hours of the file saved time stamp represents differently. Need to replace hours with regex match.
```suggestion
      "\nStart:             Nov  7 2023 .*:30:36\\..* \\(1699345836\\..*\\)"
      "\nEnd:               Nov  7 2023 .*:30:36\\..* \\(1699345836\\..*\\)"
```

Reply:
@MichaelOrlov
Yes. I also found it. So I submitted a fix [99b69d4](https://github.com/ros2/rosbag2/pull/1480/commits/99b69d45dd936ef01b99b44c2a24ee434b55852c).   
You are right. It is related to time zones.  
I will re-submit a fix.